management_api.go

```diff
@@ -160,6 +191,15 @@ func (e *Enforcer) RemoveNamedPolicy(ptype string, params ...interface{}) (bool,
 	return e.removePolicy("p", ptype, policy)
 }
 
+// RemoveNamedPolicies removes authorization rules from the current named policy.
+func (e *Enforcer) RemoveNamedPolicies(ptypes []string, rules [][] string) ([]bool, []error) {
```

Comment:
`ptypes string` is more better.  what do you think?

Reply:
What if ptype is different for some rules, in such cases we want ptype for each corresponding policy rule.

Reply:
So ```ptypes []string``` fits well.

Reply:
Agree with @nodece , we only have two ptypes: p and g. The user call it at most twice. This is OK.

Reply:
Hey @hsluoyz what about g1, I saw it in documentation, is that a valid type?
And what if casbin's supports more ptype in future. I was thinking of future scope. If its not okay, then I can change it no problem.

Reply:
So what should I do?

Reply:
Uses can only add policy of the same type.

Reply:
Agree with @nodece

Reply:
Okay, will make the changes.